src/wmtk/utils/Image.h

```diff
@@ -101,11 +101,11 @@ inline void split_and_save_3channels(const std::filesystem::path& path)
         spdlog::trace("[split_image] format doesn't support \"{}\"", path.string());
         return;
     }
-    std::filesystem::path directory = path.parent_path();
-    std::filesystem::path file = path.filename();
-    std::filesystem::path path_r(directory.string() + "/" + file.string() + "_r.exr");
-    std::filesystem::path path_g(directory.string() + "/" + file.string() + "_g.exr");
-    std::filesystem::path path_b(directory.string() + "/" + file.string() + "_b.exr");
+    const std::filesystem::path directory = path.parent_path();
+    const std::string file = path.stem().string();
+    const std::filesystem::path path_r = directory / (file + "_r.exr");
```

Comment:
I suggest to avoid creating 3 grayscale .exr files on disk and instead split them directly when loading the input file. You can use [this function](https://github.com/wildmeshing/wildmeshing-toolkit/blob/68c288d19e70cbeb41a35e6a864faf6ae6af0b48/tests/test_integral.cpp#L37-L52) to load and split them directly (I didn't want to create conflicts so I left it in the the unit tests).

Reply:
I agree. I will change that right now.

Reply:
I changed the code accordingly. I ran the _Texture Integral_ tests. They passed except for _Texture Integral Reference_ but I am unsure if it fails because of my changes.

Reply:
Can you try to merge with my changes in #274 see if it helps this unit test?

Reply:
@daniel-zint is this fixed? Once it's fixed this pull request can be merged

Reply:
yes, it is fixed